oracle/oracle_test.go

```diff
@@ -15,6 +15,7 @@ const (
 	// Minimum size supported by oracle cloud is 50 GB
 	newDiskSizeInGB = 50
 	newDiskPrefix   = "openstorage-test"
+	envKmsKeyID     = "KMS_KEY_ID"
```

Comment:
nit..  think we can keep the name of this param "kms" instead  "KMS_KEY_ID"  to keep it consistent with AWS and GCP. https://github.com/portworx/porx/blob/master/storage/hal/provider/aws/aws_storage.go#:~:text=case%20provider.KmsKey%3A 

https://github.com/portworx/porx/blob/master/storage/hal/provider/provider.go#:~:text=for%20disk%20encryption-,KmsKey%20%3D%20%22kms%22,-//%20TagsKey%20is%20a

Reply:
yes @pp511 In porx repo, for cloud-drive volume spec, we will keep it same as for other providers. This is just a test in cloudops that looks for user provided key id from env var.